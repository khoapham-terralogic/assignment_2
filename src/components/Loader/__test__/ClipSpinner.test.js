import React from 'react'
import { shallow } from 'enzyme'
import { ClipSpinner } from '..'

const props = {
    css: "",
    size: 15,
    color: "#fff"
}

describe("<ClipSpinner />", () => {
    test("It should render a default white spinner with a size of 15", () => {
        const wrapper = shallow(<ClipSpinner {...props} />)
        expect(wrapper).toMatchSnapshot();
    })
})
