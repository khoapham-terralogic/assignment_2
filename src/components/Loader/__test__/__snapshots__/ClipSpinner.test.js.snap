// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClipSpinner /> It should render a default white spinner with a size of 15 1`] = `
<div>
  <Loader
    color="#fff"
    css=""
    loading={true}
    size={15}
  />
</div>
`;
